### PR TITLE
Make a whole bunch of small user experience improvements

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+    </td>
+    <td valign="middle">
+      <a href="https://travis-ci.org/sass/sass-migrator"><img alt="Travis build status" src="https://api.travis-ci.org/sass/sass-migrator.svg?branch=master"></a>
+    </td>
+  </tr>
+</table>
 
-This is still a work-in-progress, but should eventually implement everything
-described in [sass/sass#2575](https://github.com/sass/sass/issues/2575).
+The Sass migrator automatically updates stylesheets that deprecated behavior to
```
Comment (jathak) [inline README.md]: Should this say "that contain deprecated behavior"?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+<table>
+  <tr>
+    <td>
+      <img width="118px" alt="Sass logo" src="https://rawgit.com/sass/sass-site/master/source/assets/img/logos/logo.svg" />
+    </td>
+    <td valign="middle">
+      <a href="https://www.npmjs.com/package/sass-migrator"><img width="100%" alt="npm statistics" src="https://nodei.co/npm/sass-migrator.png?downloads=true"></a>
+    </td>
+    <td valign="middle">
+      <a href="https://travis-ci.org/sass/sass-migrator"><img alt="Travis build status" src="https://api.travis-ci.org/sass/sass-migrator.svg?branch=master"></a>
```
Comment (jathak) [inline README.md]: The path for both of these URLs should be `sass/migrator`

Comment (nex3) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (nex3) [reply README.md]: Done.

**Pull request merged**